src/test/scala/com/cognite/sdk/scala/common/MockK8sServiceToken.scala

```diff
@@ -0,0 +1,12 @@
+package com.cognite.sdk.scala.common
+
+import cats.effect.Async
+
+final case class MockK8sServiceToken(success: Boolean = true) extends K8sServiceTokenProvider {
```

Comment:
As previously discussed, this class isn't useful now.

Reply:
removed :)